lib/Solver/MetaSMTSolver.cpp

```diff
@@ -17,6 +18,8 @@
 #include "klee/util/Assignment.h"
 #include "klee/util/ExprUtil.h"
 
+#include "llvm/Support/ErrorHandling.h"
+
 #include <metaSMT/DirectSolver_Context.hpp>
 #include <metaSMT/backend/Z3_Backend.hpp>
 #include <metaSMT/backend/Boolector.hpp>
```

Comment:
@hoangmle The old code had `#include <metaSMT/backend/STP.hpp>` (plus some horrifying `'#define`s and `#undef`s). Why do you not need these any more?